src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatActions.ts

```diff
@@ -283,43 +279,13 @@ registerActiveXtermAction({
 	}
 });
 
-registerActiveXtermAction({
-	id: TerminalChatCommandId.ViewInChat,
```

Comment:
Is this now redundant? We want to keep view in chat

Reply:
added it back 👍🏼